phastSim/phastSim.py

```diff
@@ -2044,7 +2045,7 @@ def writeGenomeMAT(self, node, mat, chromosome):
                     m_pb.mut_nuc.append(self.alleles[ch])
 
         for c in node.children:
-            self.writeGenomeMAT(c, mat)
+            self.writeGenomeMAT(c, mat, chromosome)
```

Comment:
Keep this change - and also delete lines 2006-2009 from this file: 

```python
        if not chromosome:
            from Bio import SeqIO
            chromosome = SeqIO.read(self.args.reference, format='fasta').id

```

Reply:
Now done in: [5100f4f](https://github.com/NicolaDM/phastSim/pull/20/commits/5100f4f42b4006ce553444ef8d2019750c8360a0)